src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,72 @@
+package commands;
+
+import functional.DukeException;
+import functional.TaskList;
+import functional.Ui;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDo;
```

Comment:
I like the way you organise your files in different packages. It's very neat to me!